benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -319,6 +342,17 @@ class KeycloakScenarioBuilder {
     this
   }
 
+  def deleteRealm(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .exec(http("Delete realm")
+        .delete(REALMS_ENDPOINT + "/${createdRealmId}")
+        .header("Authorization", "Bearer ${token}")
+        .header("Content-Type", "application/json")
+        .check(status.is(204)))
+      .exitHereIfFailed
```

Comment:
if the purpose of ${createdRealmId} is fulfilled, lets reap the memory back using 
`.exec(session => (session.removeAll("createdRealmId")))` after the `.check(status.is(204))` is performed on #351

Reply:
Indeed. Moved the `removeAll` call from `createRealm` to `deleteRealm` (as this is required by this step anyway)

Reply:
Cool, thanks.